src/main/java/roomescape/member/MemberService.java

```diff
@@ -1,17 +1,38 @@
 package roomescape.member;
 
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
+import roomescape.member.Member;
+import roomescape.token.JwtTokenService;
 
 @Service
+@RequiredArgsConstructor
 public class MemberService {
-    private MemberDao memberDao;
 
-    public MemberService(MemberDao memberDao) {
-        this.memberDao = memberDao;
-    }
+    private final MemberDao memberDao;
+    private final JwtTokenService jwtTokenService;
 
-    public MemberResponse createMember(MemberRequest memberRequest) {
+    public roomescape.member.MemberResponse createMember(MemberRequest memberRequest) {
         Member member = memberDao.save(new Member(memberRequest.getName(), memberRequest.getEmail(), memberRequest.getPassword(), "USER"));
```

Comment:
```suggestion
   Member member = memberDao.save(
                new Member(
                        memberRequest.getName(),
                        memberRequest.getEmail(), memberRequest.getPassword(),
                        "USER"));
```
열 제한에 넘어가지 않게끔 적절한 줄바꿈을 해주면 코드를 읽기가 수월할 것 같아요!